browser-test/src/admin_program_creation.test.ts

```diff
@@ -181,6 +181,18 @@ describe('program creation', () => {
     ])
   })
 
+  it('delete first block and edit', async () => {
```

Comment:
Can you add a comment why we're focusing on "delete first block". ie Why should someone later on not make it the 3rd block?

Reply:
added a comment clarifying why we're specifically testing deleting the first block